Sources/Extensions/Foundation/CalendarExtensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CalendarExtensions.swift
+//  SwifterSwift
+//
+//  Created by Chaithanya Prathyush on 09/11/17.
+//  Copyright © 2017 SwifterSwift
+//
+
+import Foundation
+
```

Comment:
Please add the following header to keep the consistency with other extensions in the library
```swift
// MARK: - Methods
```